exseas_explorer/util.py

```diff
@@ -76,6 +78,7 @@ def filter_patches(
     return df
 
 
+@functools.lru_cache
```

Comment:
```suggestion
@functools.cache
```

Can use `cache` when allowing unlimited number of entries.